packages/connector-alipay-native/src/constant.ts

```diff
@@ -0,0 +1,54 @@
+import path from 'path';
+
+import { ConnectorType, ConnectorMetadata, ConnectorPlatform } from '@logto/connector-types';
+import { getFileContents } from '@logto/shared';
+
+export const alipayEndpoint = 'https://openapi.alipay.com/gateway.do';
+export const methodForAccessToken = 'alipay.system.oauth.token';
+export const methodForUserInfo = 'alipay.user.info.share';
+
+// Constants in this segment are for SDK call
+// See: https://opendocs.alipay.com/open/218/105327
+export const staticParameters = {
+  apiname: 'com.alipay.account.auth',
```

Comment:
So alipay has a param called `apiname`?

Reply:
Generating URL for native SDK relies on some predefined constants.
You can check [here](https://opendocs.alipay.com/open/218/105327) for more context.

Reply:
The answer is 'YES'.

Reply:
SMH